ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -16,7 +16,9 @@ Also available in OpenShift docs: [IPsec encryption for external traffic](https:
 
 ### Enabling IPsec encryption
 
-To enable IPsec apply the following yaml:
+Include the `enable-ipsec.yaml` in the additional install-time manifests folder set in SiteConfig.
```

Comment:
Lets rephrase these to something like this:
(BTW is the folder install-time or sno-extra-manifest?)
 
To deploy with ZTP, include `enable-ipsec.yaml` in the additional install-time manifests folder set in SiteConfig.

Or optionally, it can be manually applied with:

Reply:
@yliu127 please note the manual steps are removed in this commit.

Reply:
Thanks for confirming. @sabinaaledort